kombu/transport/SQS.py

```diff
@@ -450,6 +454,29 @@ def visibility_timeout(self):
         return (self.transport_options.get('visibility_timeout') or
                 self.default_visibility_timeout)
 
+    @cached_property
+    def predefined_queue_urls(self):
```

Comment:
It is useful to add this to the transport documentation as well. See [here](http://docs.celeryproject.org/en/latest/getting-started/brokers/sqs.html).

Reply:
Created https://github.com/celery/celery/pull/4534 for documentation change.